packages/flutter/lib/src/material/text_theme.dart

```diff
@@ -83,14 +80,14 @@ import 'typography.dart';
 /// "medium" is `FontWeight.w500`. Letter spacing for all of the original
 /// text styles was 0.0.
 ///
-/// The old names are deprecated in this API.
-///
 /// Since the names `body1` and `body2` are used in both specifications but with
 /// different meanings, the API uses the terms `bodyText1` and `bodyText2` for
-/// the new API.
+/// the newer API.
```

Comment:
This would be a good place to explain that the current default text styles are based on the 2018 version of the Material spec and that there's an older set of text styles from the 2014 spec, that are still available for the sake of backwards compatibility [please phrase this point more clearly].